mlir/tools/mlir-miopen-driver/mlir-miopen-driver.cpp

```diff
@@ -191,32 +193,42 @@ static LogicalResult runMLIRPasses(ModuleOp &module,
         return failure();
       }
     }
+    if ((kernelPipelineSet.contains("tuning") ||
+         kernelPipelineSet.contains("applicability")) &&
+        kernelPipelineSet.size() != 1) {
+      llvm::errs() << "The `tuning` and `applicability` pipelines cannot be "
+                      "combined with any other pipeline options.\n";
+      return failure();
+    }
```

Comment:
Ok, who actually uses "tuning" and why was it just that one pass? @sjw36

Reply:
Having looked, `tuning` is used only in some of the mlir-miopen-lib tests, where it could probably be replaced by calling `-miopen-affix-params` directly

Reply:
(and so that's the question for you, Jerry)

Reply:
Well I can also argue that the creation of `applicability` option is unnecessary. But I don't feel strong to hold back the review because of this minor refactoring item.

Reply:
I think what I'll do here is get rid of the `tuning` option.

Reply:
Sounds good to me